ubuntu-github-actions--20.04/Dockerfile

```diff
@@ -70,7 +74,10 @@ RUN \
     libdc1394-22-dev \
     libgphoto2-dev \
     && \
-  mkdir /opencv /opencv_contrib /opencv-build /opencv-contrib-build /build && \
+  mkdir /opencv /opencv_contrib /build && \
+  git config --global --add safe.directory "*" && \
```

Comment:
It works only in the latest versions of git ([update](https://marc.info/?l=git&m=164989570902912&w=2)).